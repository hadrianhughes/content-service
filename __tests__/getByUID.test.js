const getByUID = require('../lib/getByUID');

const mockData = require('../__mock-data__/doc.json');
const mockApi = {
  getByUID: string => new Promise ((resolve, reject) => {
    resolve(mockData);
  })
};

describe('getByUID function', () => {
  it('Should return a promise which resolves with an object', () => {
    expect(getByUID(mockApi, 'foo').resolves).toMatchSnapshot();
  });
});
