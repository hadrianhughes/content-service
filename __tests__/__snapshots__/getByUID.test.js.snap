// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getByUID function Should return a promise which resolves with an object 1`] = `undefined`;
